Makefile

```
@@ -64,15 +68,14 @@ inspect_mcp:
 	MCPI_NO_COLOR=1 RUST_LOG=warn RUSTFLAGS="-A warnings" npx @modelcontextprotocol/inspector ./target/debug/dela mcp --cwd $(PWD)
 
 
-# Publish to crates.io
+# Publish to crates.io. Deprecated. Will be removed once 0.0.7 is released sucessfully using the new release process.
 publish: tests tests_integration
```

checkmake (0.3.2)

[warning] Target body for "publish" exceeds allowed length of 5 lines (6).

(maxbodylength)